sway/config.c

```diff
@@ -620,23 +621,127 @@ bool load_include_configs(const char *path, struct sway_config *config) {
 	return true;
 }
 
+/***************************************************************************//**
+*
+*	\brief	This function corrects the given 'line' if the opening parentheses
+*	happens to be on the next line.
+*
+*	\param[in]	file The pointer to the already open file.
+* \param[in/out]	line The current line read from the configuration file.
+* \param[in]	line_number Counter for the line number. Optional.
+*
+*	\return	A string to the corrected 'line' is returned.
+*
+*******************************************************************************/
+static inline char* handle_parentheses(FILE* const file, char* line,
```

Comment:
Technically the word is braces, not parenthesis. () are parenthesis, {} are braces.